plugins/org.jkiss.dbeaver.model.ai/src/org/jkiss/dbeaver/model/ai/utils/AIUtils.java

```diff
@@ -141,4 +156,74 @@ public static String processCompletion(
 
         return formatter.postProcessGeneratedQuery(monitor, mainObject, executionContext, completionText).trim();
     }
+
+    /**
+     * Checks if the given DBPObject is eligible for AI description.
+     *
+     * @param dbpObject the object to check
+     * @return true if the object can be described by AI, false otherwise
+     */
+    public static boolean isEligible(@Nullable DBPObject dbpObject) {
```

Comment:
Why are these here? Shouldn't they be in the other repo?

Reply:
No